src/main/java/com/minecolonies/coremod/commands/ActionMenuType.java

```diff
@@ -0,0 +1,300 @@
+import com.minecolonies.coremod.commands.colonycommands.ListColoniesCommand;
+import com.minecolonies.coremod.commands.colonycommands.RefreshColonyCommand;
+import com.minecolonies.coremod.commands.colonycommands.ShowColonyInfoCommand;
+import com.minecolonies.coremod.commands.colonycommands.requestsystem.RSResetAllCommand;
+import com.minecolonies.coremod.commands.colonycommands.requestsystem.RSResetCommand;
+import com.minecolonies.coremod.commands.generalcommands.BackupCommand;
+import com.minecolonies.coremod.commands.generalcommands.CheckForAutoDeletesCommand;
+import com.minecolonies.coremod.commands.generalcommands.RaidAllNowCommand;
+import com.minecolonies.coremod.commands.generalcommands.RaidAllTonightCommand;
+import com.minecolonies.coremod.commands.generalcommands.RandomTeleportCommand;
+import com.minecolonies.coremod.commands.generalcommands.ScanCommand;
+import com.minecolonies.coremod.commands.generalcommands.WhereAmICommand;
+import com.minecolonies.coremod.commands.generalcommands.WhoAmICommand;
+import com.minecolonies.coremod.commands.killcommands.AnimalKillCommand;
+import com.minecolonies.coremod.commands.killcommands.BarbarianKillCommand;
+import com.minecolonies.coremod.commands.killcommands.ChickenKillCommand;
+import com.minecolonies.coremod.commands.killcommands.CowKillCommand;
+import com.minecolonies.coremod.commands.killcommands.MobKillCommand;
+import com.minecolonies.coremod.commands.killcommands.PigKillCommand;
+import com.minecolonies.coremod.commands.killcommands.SheepKillCommand;
+
+public enum ActionMenuType implements MenuType
+{
+    RandomTeleport(new ActionMenu(
+            "Random Teleport",
+            "rtp",
+            ForgePermissionNodes.RandomTeleport,
+            RandomTeleportCommand.class,
+            new ActionArgument("player", ActionArgumentType.Player, ActionArgumentType.Is.Optional)
+            )),
+    Backup(new ActionMenu(
+            "Backup",
+            "backup",
+            ForgePermissionNodes.Backup,
+            BackupCommand.class
+            )),
+    HomeTeleport(new ActionMenu(
+            "HomeTeleport",
+            "home",
+            ForgePermissionNodes.HomeTeleport,
+            HomeTeleportCommand.class
+            )),
+    RaidAllTonight(new ActionMenu(
+            "RaidAllTonight",
+            "raid-tonight",
+            ForgePermissionNodes.RaidAllTonight,
+            RaidAllTonightCommand.class
+            )),
+    RaidAllNow(new ActionMenu(
+            "RaidAllNow",
+            "raid-now",
+            ForgePermissionNodes.RaidAllNow,
+            RaidAllNowCommand.class
+            )),
+    CheckForAutoDeletes(new ActionMenu(
+            "CheckForAutoDeletes",
+            "check",
+            ForgePermissionNodes.CheckForAutoDeletes,
+            CheckForAutoDeletesCommand.class,
+            new ActionArgument("confirmDelete", ActionArgumentType.BOOLEAN, ActionArgumentType.Is.Optional)
+            )),
+    WhoAmI(new ActionMenu(
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00115)